### ISSUE TITLE
emacs (doom) and shell (zplug) did not update 

### ISSUE THREAD

**Issue by jeetelongname** [r-darwish/topgrade#553]
**What did you expect to happen?**
doom and zplug would update when I moved them to either there xdg compilent
place or to another place 
I belive this is because these paths are hardcoded. zplug does provide the
variable `$ZPLUG_HOME` which shows the location of zplug if thats any help 
and emacs is in `.config/emacs` and doom is in `.config/doom`
**What actually happened?**
... this did not happen

**Additional details:**
- Which operating system or Linux distribution are you using?
  Pop os
- How did you install Topgrade?
  through `cargo`
- If a specific step is failing, what happens when you run the command on your
  own? (Run `topgrade --dry-run` to see which commands Topgrade is running)
  I ran `topgrade --dry-run --only emacs shell`
  
`(R)eboot
(S)hell
(Q)uit
`<details><pre>
chatterlap:(Tue 3) ᐅ topgrade -v 
 2020-11-03T18:06:01.752Z DEBUG topgrade::config > Configuration at /home/jeet/.config/topgrade.toml
 2020-11-03T18:06:01.756Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: None, post_commands: None, commands: None, git_repos: None, predefined_git_repos: None, disable: Some([Go, GitRepos, Myrepos]), ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, assume_yes: Some(true), yay_arguments: None, no_retry: None, run_in_tmux: Some(true), cleanup: None, notify_each_step: None, accept_all_windows_updates: None, only: None, composer: None, brew: None, linux: None, git: None, windows: None, vagrant: None }
 2020-11-03T18:06:01.757Z DEBUG topgrade::utils  > Detected "/usr/bin/notify-send" as "notify-send"
 2020-11-03T18:06:01.757Z DEBUG topgrade         > Version: 5.8.1
 2020-11-03T18:06:01.757Z DEBUG topgrade         > OS: x86_64-unknown-linux-gnu
 2020-11-03T18:06:01.757Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2020-11-03T18:06:01.757Z DEBUG topgrade         > Binary path: Ok("/home/jeet/.local/share/cargo/bin/topgrade")
 2020-11-03T18:06:01.757Z DEBUG topgrade         > Self Update: false
 2020-11-03T18:06:01.757Z DEBUG topgrade::utils  > Detected "/usr/bin/tmux" as "tmux"
 Topgrade launched in a new tmux session
P 2020-11-03T18:06:01.776Z DEBUG topgrade::config > Configuration at /home/jeet/.config/topgrade.toml
 2020-11-03T18:06:01.776Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: None, post_commands: None, commands: None, git_repos: None, predefined_git_repos: None, disable: Some([Go, GitRepos, Myrepos]), ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, assume_yes: Some(true), yay_arguments: None, no_retry: None, run_in_tmux: Some(true), cleanup: None, notify_each_step: None, accept_all_windows_updates: None, only: None, composer: None, brew: None, linux: None, git: None, windows: None, vagrant: None }
 2020-11-03T18:06:01.776Z DEBUG topgrade::utils  > Detected "/usr/bin/notify-send" as "notify-send"
 2020-11-03T18:06:01.776Z DEBUG topgrade         > Version: 5.8.1
 2020-11-03T18:06:01.776Z DEBUG topgrade         > OS: x86_64-unknown-linux-gnu
 2020-11-03T18:06:01.776Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2020-11-03T18:06:01.776Z DEBUG topgrade         > Binary path: Ok("/home/jeet/.local/share/cargo/bin/topgrade")
 2020-11-03T18:06:01.776Z DEBUG topgrade         > Self Update: false
 2020-11-03T18:06:01.776Z DEBUG topgrade::utils  > Detected "/usr/bin/git" as "git"
 2020-11-03T18:06:01.776Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"
 2020-11-03T18:06:01.776Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2020-11-03T18:06:01.776Z DEBUG topgrade::utils  > Cannot find "powershell"
 2020-11-03T18:06:01.776Z DEBUG topgrade::runner > Step "System update"
―― 18:06:01 - System update ――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-03T18:06:01.777Z DEBUG topgrade::utils  > Cannot find "apt-fast"
 2020-11-03T18:06:01.777Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "/usr/bin/apt" "update"
[sudo] password for jeet:
Hit:1 http://ppa.launchpad.net/system76/pop/ubuntu groovy InRelease
Hit:2 http://us.archive.ubuntu.com/ubuntu groovy InRelease
Hit:3 http://us.archive.ubuntu.com/ubuntu groovy-security InRelease
Hit:4 http://apt.pop-os.org/proprietary groovy InRelease
Hit:5 http://us.archive.ubuntu.com/ubuntu groovy-updates InRelease
Hit:6 http://us.archive.ubuntu.com/ubuntu groovy-backports InRelease
Reading package lists... Done
Building dependency tree
Reading state information... Done
All packages are up-to-date.
 2020-11-03T18:06:48.081Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "/usr/bin/apt" "dist-upgrade" "-y"
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
The following package was automatically installed and is no longer required:
  libre2-5
Use 'sudo apt autoremove' to remove it.
0 to upgrade, 0 to newly install, 0 to remove and 0 not to upgrade.
 2020-11-03T18:06:51.741Z DEBUG topgrade::runner   > Step "etc-update"
 2020-11-03T18:06:51.741Z DEBUG topgrade::runner   > Step "Brew"
 2020-11-03T18:06:51.741Z DEBUG topgrade::runner   > Step "yadm"
 2020-11-03T18:06:51.741Z DEBUG topgrade::runner   > Step "nix"
 2020-11-03T18:06:51.741Z DEBUG topgrade::runner   > Step "home-manager"
 2020-11-03T18:06:51.741Z DEBUG topgrade::runner   > Step "asdf"
 2020-11-03T18:06:51.742Z DEBUG topgrade::utils    > Path "/home/jeet/.config/emacs" exists
 2020-11-03T18:06:51.742Z DEBUG topgrade::utils    > Path "/home/jeet/.emacs.d" exists
 2020-11-03T18:06:51.742Z DEBUG topgrade::utils    > Path "/home/jeet/.emacs.d/bin/doom" doesn't exist
 2020-11-03T18:06:51.742Z DEBUG topgrade::steps::git > Checking if /home/jeet/.emacs.d is a git repository
 2020-11-03T18:06:51.744Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2020-11-03T18:06:51.744Z DEBUG topgrade::steps::git > /home/jeet/.doom.d does not exists
 2020-11-03T18:06:51.744Z DEBUG topgrade::steps::git > /home/jeet/.vim does not exists
 2020-11-03T18:06:51.744Z DEBUG topgrade::steps::git > Checking if /home/jeet/.dotfiles/neovim/.config/nvim is a git repository
 2020-11-03T18:06:51.746Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/jeet/.dotfiles\n", stderr: "" }
 2020-11-03T18:06:51.746Z DEBUG topgrade::steps::git > /home/jeet/.dotfiles/shell/.config/zsh/.zshrc is a file. Checking /home/jeet/.dotfiles/shell/.config/zsh
 2020-11-03T18:06:51.746Z DEBUG topgrade::steps::git > Checking if /home/jeet/.dotfiles/shell/.config/zsh is a git repository
 2020-11-03T18:06:51.748Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: cannot chdir to \'../../../../../zsh/.config/zsh\': No such file or directory\n" }
 2020-11-03T18:06:51.748Z DEBUG topgrade::steps::git > Checking if /home/jeet/.tmux is a git repository
 2020-11-03T18:06:51.750Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2020-11-03T18:06:51.750Z DEBUG topgrade::steps::git > /home/jeet/.config/fish does not exists
 2020-11-03T18:06:51.750Z DEBUG topgrade::steps::git > /home/jeet/.config/openbox does not exists
 2020-11-03T18:06:51.750Z DEBUG topgrade::steps::git > /home/jeet/.config/bspwm does not exists
 2020-11-03T18:06:51.750Z DEBUG topgrade::steps::git > /home/jeet/.config/i3 does not exists
 2020-11-03T18:06:51.750Z DEBUG topgrade::steps::git > /home/jeet/.config/sway does not exists
 2020-11-03T18:06:51.750Z DEBUG topgrade::runner     > Step "zr"
 2020-11-03T18:06:51.750Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-03T18:06:51.750Z DEBUG topgrade::runner     > Step "antibody"
 2020-11-03T18:06:51.750Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-03T18:06:51.750Z DEBUG topgrade::runner     > Step "antigen"
 2020-11-03T18:06:51.750Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-03T18:06:51.750Z DEBUG topgrade::utils      > Path "/home/jeet/.config/zsh/.zshrc" exists
 2020-11-03T18:06:51.750Z DEBUG topgrade::runner     > Step "zplug"
 2020-11-03T18:06:51.750Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-03T18:06:51.750Z DEBUG topgrade::utils      > Path "/home/jeet/.config/zsh/.zshrc" exists
 2020-11-03T18:06:51.750Z DEBUG topgrade::runner     > Step "zinit"
 2020-11-03T18:06:51.750Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-03T18:06:51.750Z DEBUG topgrade::utils      > Path "/home/jeet/.config/zsh/.zshrc" exists
 2020-11-03T18:06:51.750Z DEBUG topgrade::runner     > Step "oh-my-zsh"
 2020-11-03T18:06:51.750Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-03T18:06:51.750Z DEBUG topgrade::runner     > Step "fisher"
 2020-11-03T18:06:51.751Z DEBUG topgrade::runner     > Step "oh-my-fish"
 2020-11-03T18:06:51.751Z DEBUG topgrade::runner     > Step "tmux"
 2020-11-03T18:06:51.751Z DEBUG topgrade::utils      > Path "/home/jeet/.tmux/plugins/tpm/bin/update_plugins" exists
―― 18:06:51 - tmux plugins ―――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-03T18:06:51.751Z DEBUG topgrade::executor   > Running "/home/jeet/.tmux/plugins/tpm/bin/update_plugins" "all"
Updating all plugins!

  "tmux-prefix-highlight" update success
  "tmux-battery" update success
  "tmux-yank" update success
  "tmux-open" update success
  "tpm" update success
 2020-11-03T18:06:54.042Z DEBUG topgrade::runner     > Step "TLDR"
 2020-11-03T18:06:54.042Z DEBUG topgrade::utils      > Detected "/home/jeet/code/go/bin/tldr" as "tldr"
―― 18:06:54 - TLDR ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-03T18:06:54.043Z DEBUG topgrade::executor   > Running "/home/jeet/code/go/bin/tldr" "--update"
 2020-11-03T18:06:55.229Z DEBUG topgrade::runner     > Step "pearl"
 2020-11-03T18:06:55.229Z DEBUG topgrade::runner     > Step "SDKMAN!"
 2020-11-03T18:06:55.229Z DEBUG topgrade::utils      > Detected "/usr/bin/bash" as "bash"
 2020-11-03T18:06:55.229Z DEBUG topgrade::runner     > Step "apm"
 2020-11-03T18:06:55.229Z DEBUG topgrade::runner     > Step "rustup"
 2020-11-03T18:06:55.229Z DEBUG topgrade::runner     > Step "choosenim"
 2020-11-03T18:06:55.229Z DEBUG topgrade::runner     > Step "cargo"
 2020-11-03T18:06:55.230Z DEBUG topgrade::runner     > Step "Flutter"
 2020-11-03T18:06:55.230Z DEBUG topgrade::runner     > Step "Emacs"
 2020-11-03T18:06:55.230Z DEBUG topgrade::utils      > Detected "/usr/local/bin/emacs" as "emacs"
 2020-11-03T18:06:55.230Z DEBUG topgrade::runner     > Step "opam"
 2020-11-03T18:06:55.230Z DEBUG topgrade::runner     > Step "vcpkg"
 2020-11-03T18:06:55.230Z DEBUG topgrade::runner     > Step "pipx"
 2020-11-03T18:06:55.230Z DEBUG topgrade::runner     > Step "stack"
 2020-11-03T18:06:55.230Z DEBUG topgrade::runner     > Step "tlmgr"
 2020-11-03T18:06:55.230Z DEBUG topgrade::runner     > Step "jetpack"
 2020-11-03T18:06:55.230Z DEBUG topgrade::runner     > Step "vim"
 2020-11-03T18:06:55.230Z DEBUG topgrade::utils      > Detected "/usr/bin/vim" as "vim"
 2020-11-03T18:06:55.237Z TRACE topgrade::executor   > Output of "/usr/bin/vim" "--version": Output { status: ExitStatus(ExitStatus(0)), stdout: "NVIM v0.4.4\nBuild type: Release\nLuaJIT 2.1.0-beta3\nCompilation: /usr/bin/cc -g -O2 -fdebug-prefix-map=/build/neovim-u4YhjF/neovim-0.4.4=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=1 -DDISABLE_LOG -Wdate-time -D_FORTIFY_SOURCE=1 -O2 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/build/neovim-u4YhjF/neovim-0.4.4/build/config -I/build/neovim-u4YhjF/neovim-0.4.4/src -I/usr/include -I/usr/include/lua5.1 -I/build/neovim-u4YhjF/neovim-0.4.4/build/src/nvim/auto -I/build/neovim-u4YhjF/neovim-0.4.4/build/include\nCompiled by team+vim@tracker.debian.org\n\nFeatures: +acl +iconv +tui\nSee \":help feature-compile\"\n\n   system vimrc file: \"$VIM/sysinit.vim\"\n  fall-back for $VIM: \"/usr/share/nvim\"\n\nRun :checkhealth for more info\n", stderr: "" }
 2020-11-03T18:06:55.237Z DEBUG topgrade::runner     > Step "Neovim"
 2020-11-03T18:06:55.237Z DEBUG topgrade::utils      > Detected "/usr/bin/nvim" as "nvim"
 2020-11-03T18:06:55.237Z DEBUG topgrade::utils      > Path "/home/jeet/.config/nvim/init.vim" exists
―― 18:06:55 - Neovim ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-03T18:06:55.238Z DEBUG topgrade::steps::vim > Wrote vim script to "/tmp/.tmpkH1sPQ"
not found in 'packpath': "pack/*/start/*"
not found in 'packpath': "pack/*/start/matchit"
not found in 'packpath': "pack/*/start/*"
not found in 'runtimepath': "plugin/**/*.vim"
Reading ShaDa file "/home/jeet/.local/share/nvim/shada/main.shada" info marks oldfiles
Plug
Downloading the latest version of vim-plugvim-plug is already up-to-date
not found in 'runtimepath': "ftplugin/vim-plug.vim ftplugin/vim-plug_*.vim ftplugin/vim-plug/*.vim"
not found in 'runtimepath': "indent/vim-plug.vim"
not found in 'runtimepath': "syntax/vim-plug.vim syntax/vim-plug/*.vim"Writing ShaDa file "/home/jeet/.local/share/nvim/shada/main.shada"Plugins upgraded
 2020-11-03T18:06:58.479Z DEBUG topgrade::runner     > Step "voom"
 2020-11-03T18:06:58.479Z DEBUG topgrade::runner     > Step "npm"
 2020-11-03T18:06:58.479Z DEBUG topgrade::utils      > Detected "/usr/bin/npm" as "npm"
 2020-11-03T18:06:58.885Z TRACE topgrade::executor   > Output of "/usr/bin/npm" "root" "-g": Output { status: ExitStatus(ExitStatus(0)), stdout: "/usr/local/lib/node_modules\n", stderr: "" }
 2020-11-03T18:06:58.885Z DEBUG topgrade::runner     > Step "yarn"
 2020-11-03T18:06:58.885Z DEBUG topgrade::runner     > Step "deno"
 2020-11-03T18:06:58.885Z DEBUG topgrade::runner     > Step "composer"
 2020-11-03T18:06:58.885Z DEBUG topgrade::runner     > Step "krew"
 2020-11-03T18:06:58.885Z DEBUG topgrade::runner     > Step "gem"
 2020-11-03T18:06:58.886Z DEBUG topgrade::utils      > Detected "/usr/bin/gem" as "gem"
 2020-11-03T18:06:58.886Z DEBUG topgrade::runner     > Step "sheldon"
 2020-11-03T18:06:58.886Z DEBUG topgrade::runner     > Step "rtcl"
 2020-11-03T18:06:58.886Z DEBUG topgrade::runner     > Step "Flatpak"
 2020-11-03T18:06:58.886Z DEBUG topgrade::utils      > Detected "/usr/bin/flatpak" as "flatpak"
―― 18:06:58 - Flatpak User Packages ――――――――――――――――――――――――――――――――――――――――――――
 2020-11-03T18:06:58.886Z DEBUG topgrade::executor   > Running "/usr/bin/flatpak" "update" "--user" "-y"
Looking for updates…
Nothing to do.
 2020-11-03T18:06:59.468Z DEBUG topgrade::executor   > Running "/usr/bin/flatpak" "update" "--system" "-y"
Looking for updates…
Nothing to do.
 2020-11-03T18:06:59.558Z DEBUG topgrade::runner     > Step "snap"
 2020-11-03T18:06:59.558Z DEBUG topgrade::utils      > Detected "/usr/bin/snap" as "snap"
―― 18:06:59 - snap ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-03T18:06:59.558Z DEBUG topgrade::executor   > Running "/usr/bin/sudo" "/usr/bin/snap" "refresh"
All snaps up to date.
 2020-11-03T18:07:00.232Z DEBUG topgrade::runner     > Step "pihole"
 2020-11-03T18:07:00.233Z DEBUG topgrade::runner     > Step "Firmware upgrades"
 2020-11-03T18:07:00.233Z DEBUG topgrade::utils      > Detected "/usr/bin/fwupdmgr" as "fwupdmgr"
 2020-11-03T18:07:00.234Z TRACE topgrade::executor   > Output of "uname" "-r": Output { status: ExitStatus(ExitStatus(0)), stdout: "5.8.0-7625-generic\n", stderr: "" }
 2020-11-03T18:07:00.234Z DEBUG topgrade::steps::os::linux > Uname output: 5.8.0-7625-generic
―― 18:07:00 - Firmware upgrades ――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-03T18:07:00.234Z DEBUG topgrade::executor         > Running "/usr/bin/fwupdmgr" "refresh"
Firmware metadata last refresh: 1 hour ago. Use --force to refresh again.
 2020-11-03T18:07:00.275Z DEBUG topgrade::executor         > Running "/usr/bin/fwupdmgr" "get-updates"
• HFM256GDJTNG-8310A has no available firmware updates
• System Firmware has no available firmware updates
No updatable devices
 2020-11-03T18:07:00.314Z DEBUG topgrade::runner           > Step "Restarts"
―― 18:07:00 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
System update: OK
etc-update: SKIPPED: Cannot find "etc-update" in PATH
Brew: SKIPPED: Cannot find "brew" in PATH
yadm: SKIPPED: Cannot find "yadm" in PATH
nix: SKIPPED: Cannot find "nix" in PATH
home-manager: SKIPPED: Cannot find "home-manager" in PATH
asdf: SKIPPED: Cannot find "asdf" in PATH
zr: SKIPPED: Cannot find "zr" in PATH
antibody: SKIPPED: Cannot find "antibody" in PATH
antigen: SKIPPED: Path "/home/jeet/antigen.zsh" doesn't exist
zplug: SKIPPED: Path "/home/jeet/zplug" doesn't exist
zinit: SKIPPED: Path "/home/jeet/.zinit" doesn't exist
oh-my-zsh: SKIPPED: Path "/home/jeet/.oh-my-zsh" doesn't exist
fisher: SKIPPED: Cannot find "fish" in PATH
oh-my-fish: SKIPPED: Cannot find "fish" in PATH
tmux: OK
TLDR: OK
pearl: SKIPPED: Cannot find "pearl" in PATH
SDKMAN!: SKIPPED: Path "/home/jeet/.sdkman/bin/sdkman-init.sh" doesn't exist
apm: SKIPPED: Cannot find "apm" in PATH
rustup: SKIPPED: Cannot find "rustup" in PATH
choosenim: SKIPPED: Cannot find "choosenim" in PATH
cargo: SKIPPED: Cannot find "cargo-install-update" in PATH
Flutter: SKIPPED: Cannot find "flutter" in PATH
Emacs: SKIPPED: Path "/home/jeet/.emacs.d/init.el" doesn't exist
opam: SKIPPED: Cannot find "opam" in PATH
vcpkg: SKIPPED: Cannot find "vcpkg" in PATH
pipx: SKIPPED: Cannot find "pipx" in PATH
stack: SKIPPED: Cannot find "stack" in PATH
tlmgr: SKIPPED: tlmgr must be explicity enabled in the configuration to run in Linux
jetpack: SKIPPED: Cannot find "jetpack" in PATH
vim: SKIPPED: vim binary might by actually nvim
Neovim: OK
voom: SKIPPED: Cannot find "voom" in PATH
npm: SKIPPED: NPM root at /usr/local/lib/node_modules
 isn't a decandent of the user's home directory
yarn: SKIPPED: Cannot find "yarn" in PATH
deno: SKIPPED: Cannot find "deno" in PATH
composer: SKIPPED: Cannot find "composer" in PATH
krew: SKIPPED: Cannot find "kubectl-krew" in PATH
gem: SKIPPED: Path "/home/jeet/.gem" doesn't exist
sheldon: SKIPPED: Cannot find "sheldon" in PATH
rtcl: SKIPPED: Cannot find "rupdate" in PATH
Flatpak: OK
snap: OK
pihole: SKIPPED: Cannot find "pihole" in PATH
Firmware upgrades: OK
Restarts: SKIPPED: Cannot find "needrestart" in PATH
(R)eboot
(S)hell
(Q)uit
</pre></details>

**Comment by r-darwish** [comment]
Does the file `~/.config/emacs/init.el` exists?

**Comment by jeetelongname** [comment]
yes.

**Comment by r-darwish** [comment]
You also seems to have `/home/jeet/.emacs.d` on your machine. If you use `~/.config/emacs` then can you please try to delete `/home/jeet/.emacs.d` and see if it fixes the issue with Emacs?

As for zplug, it seems like a bug in Topgrade. I'll fix it.

**Comment by jeetelongname** [comment]
I do have a .emacs that I use for [chemacs](https://github.com/plexus/chemacs) that may be affecting it

**Comment by r-darwish** [comment]
Topgrade doesn't check for `~/.emacs`, only for either `~/.emacs.d` and `~/.config/emacs`. It seems that you have both but `/home/jeet/.emacs.d/init.el` doesn't exist. This is why Topgrade skips upgrading Emacs.

**Comment by jeetelongname** [comment]
Yeah that fixes emacs. thanks!

**Comment by r-darwish** [comment]
Great. Can you please compile and run #555 to see if that fixes zplug?

**Comment by jeetelongname** [comment]
Yep that seems to fix it. thanks

**Comment by jeetelongname** [comment]
Closing now
